### ISSUE TITLE
should be 3 contributors: private project or branch with limited permissions. 

### ISSUE THREAD

**Issue by douglasdconklin** [conklinbrendan/Mud-Project#2]
it bothers me that the invite to be a contributor didn't work. 
It's worrisome  that apparently random people (me) can push any random stuff to the repository.

**Comment by grobertconklin** [comment]
yes I agree.  I will try to set it up Brendan as a paying customer of Github so he can make project private.
